bip-0001.mediawiki

```diff
@@ -1,13 +1,11 @@
 <pre>
   BIP: 1
   Title: BIP Purpose and Guidelines
-  Author: Amir Taaki <genjix@riseup.net>
-  Comments-Summary: No comments yet.
-  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0001
-  Status: Replaced
+  Authors: Amir Taaki <genjix@riseup.net>
```

Comment:
Looks like `scripts/buildtable.pl` will need to be updated for the BIP3 headers changes.

Reply:
e.g. update the required fields, Author -> Authors, remove the comments, update title length, update the licenses, update %MiscField, add checks for new fields like Deputies, etc.

Reply:
Thanks yes, I’m working on that today.

Reply:
@jonatack: This should be done